app/components/anchor/action_menu_component.rb

```diff
@@ -1,22 +1,24 @@
 module Anchor
   class ActionMenuComponent < Component
-    renders_one :show_button, -> do
+    renders_one :show_button, lambda { |**kwargs|
       ButtonComponent.new(
-        data: { action_menu_target: "button" },
-        popovertarget: @menu_id
+        data: { action_menu_target: 'button' },
+        popovertarget: @menu_id,
+        **kwargs
       )
-    end
+    }
     renders_many :items
 
-    def initialize
+    def initialize(popover_classes: '', **kwargs)
       @menu_id = self.class.generate_id
+      @popover_classes = popover_classes
```

Comment:
Thoughts on naming this `menu_classes` to match `menu_id`?

```suggestion
      @menu_classes = menu_classes
```